src/landOnFunctions.js

```diff
@@ -1,57 +1,67 @@
-export {standardProperty, railroad, utility, eventCard, noEvent, go, goToJail, incomeTax, luxuryTax};
+export {
+    standardProperty,
+    railroad,
+    utility,
+    eventCard,
+    noEvent,
+    go,
+    goToJail,
+    incomeTax,
+    luxuryTax,
+};
 
 function standardProperty(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
+        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
+    }
+    else{
+        emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[this.houses]);
+    }
 
-  return this;
+    return this;
 }
 
 function railroad(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
+        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
```

Comment:
I'll have to look over where all I used it.  I think I started using it more because I am also trying to make it harder for other player to unintentionally or maliciously spam events and hit somebody else's.

Though, in this case the reason is because the listeners are turned on when a client connects with socket.io in main.js  As I understand it, each socket.on that opens is specific to that clients open socket  Since I am turning on listeners in the Monopoly.emitter object, it would otherwise turn on a listener each time a client connects and was causing issues.  Now, it turns on a client specific listener for each client.  Just as each client has their own specific socket listeners, they have their own monopoly listeners.  It also helps the server from taking in bad events emitted by clients as mentioned earlier.